website/docs/getting-started/contribution/index.md

```diff
@@ -1,4 +1,26 @@
-## Contributing
+---
+title: Contribution
+---
+
+## Overview
+
+While we don't accept direct contributions of new components or patterns, we do welcome contributions in the following forms:
+
+* [Reporting bugs](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform) and [giving feedback](https://docs.google.com/forms/d/e/1FAIpQLSdPShbSiUYgimppZeqWT99bVi6zLbKDThQUQTirH3RCj2okdg/viewform)
+* [Proposing new components or patterns](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform)
```

Comment:
I'm not sure if I did the anchor link correctly.

```suggestion
* [Proposing new components or patterns](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform)
* [Add an icon](#iconcontributions)
```

Reply:
I accepted this without looking closely enough. I don't think this will work but I'll try to figure it out.

Reply:
`#icon-contributions` was what worked

Reply:
Perfect! Thanks for confirming; I was seeing conflicting information on the google.